presto-native-execution/scripts/dockerfiles/centos-dependency.dockerfile

```diff
@@ -19,8 +19,4 @@ ENV CXX=/opt/rh/gcc-toolset-12/root/bin/g++
 RUN mkdir -p /scripts /velox/scripts
 COPY scripts /scripts
 COPY velox/scripts /velox/scripts
-RUN mkdir build && \
-    (cd build && ../scripts/setup-centos.sh && \
-                 ../velox/scripts/setup-adapters.sh && \
-                 ../scripts/setup-adapters.sh ) && \
-    rm -rf build
+RUN chmod +x /scripts/install_minimal_dependencies.sh && /scripts/install_minimal_dependencies.sh centos
```

Comment:
Is the 644 for the script on purpose? Why don't we give it execute (755) when it is added in the first place and don't need the chmod?

The name of the script - minimal_dependencies sounds like the minimum number of dependencies to build when in fact we install everything. Minimal refers to the size of the resulting image. Trying to think of a better name because this might cause confusion when looking only at the name.

Maybe something with install_and_reduce_size.sh or something similar?